classes/class-wc-connect-shipping-label.php

```diff
@@ -287,13 +305,14 @@ public function meta_box( $post ) {
 			) );
 
 			$admin_array = array(
-				'storeOptions' => $this->settings_store->get_shared_settings(),
-				'formSchema'   => $this->get_form_schema(),
-				'formLayout'   => $this->get_form_layout(),
-				'formData'     => $this->get_form_data( $order ),
-				'callbackURL'  => get_rest_url( null, "/wc/v1/connect/shipping-label" ),
-				'nonce'        => wp_create_nonce( 'wp_rest' ),
-				'rootView'     => 'shipping-label',
+				'storeOptions'  => $this->settings_store->get_shared_settings(),
+				'formSchema'    => $this->get_form_schema(),
+				'formLayout'    => $this->get_form_layout(),
+				'formData'      => $this->get_form_data( $order ),
+				'countriesData' => $this->get_states_map(),
```

Comment:
Instead of calling this countriesData, could we be more specific and call it countriesStatesMap